.github/workflows/service_test_hdfs.yml

```diff
@@ -101,11 +101,39 @@ jobs:
           OPENDAL_HDFS_NAME_NODE: hdfs://localhost:8020
           OPENDAL_HDFS_ENABLE_APPEND: true
 
-  hdfs-default-with-atomic-write-dir:
+  hdfs-cluster-with-atomic-write-dir:
```

Comment:
Hi, please do not modify `.github/workflows/service_test_hdfs.yml`. Instead, add a new test in https://github.com/apache/opendal/tree/main/.github/services/hdfs.

Welcome to migrate existing one to `services/hdfs` too if you are interested. I forgot why those tests weren't migrated...

Reply:
ok